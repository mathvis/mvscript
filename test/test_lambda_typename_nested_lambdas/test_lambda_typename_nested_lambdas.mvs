let a: lambda[lambda[lambda[int]int]int]lambda[int]int;
