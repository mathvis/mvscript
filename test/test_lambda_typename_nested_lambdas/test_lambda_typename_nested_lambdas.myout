Decl (Variable (VarIdentifier "a") (Just (LambdaT [LambdaT [LambdaT [IntT] IntT] IntT] (LambdaT [IntT] IntT))) Nothing)
