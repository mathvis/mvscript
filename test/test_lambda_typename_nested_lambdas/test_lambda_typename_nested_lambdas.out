Decl (Variable (VarIdentifier "a") (Just lambda[lambda[lambda[int]int]int]lambda[int]int) Nothing)
